### PR TITLE
[Metricbeat] [AWS] Add DynamoDB AWS light module

### PR DESCRIPTION
**Pull request by ChrsMark**
This PR implements `DynamoDB` AWS metricbeat Module of https://github.com/elastic/beats/issues/14856

Reference: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/metrics-dimensions.html

Dashboard:
Screenshot 2020-01-08 at 12 10 39

### CODE REVIEW

**Comment by ChrsMark** [comment]
> Nice dashboard 👍 How about adding filters for regions and account name in the dashboard?

Hey! I updated the dashboard with the filters! Thanks for the suggestion!

File: x-pack/metricbeat/module/aws/dynamodb/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+default: true
+input:
+  module: aws
+  metricset: cloudwatch
+  defaults:
+    metrics:
+      - namespace: AWS/DynamoDB
+        statistic: ["Average"]
```
Comment (kaiyan-sheng) [inline x-pack/metricbeat/module/aws/dynamodb/manifest.yml]: I believe AWS allows users to add tags for DynamoDB, so we probably should add `tags.resource_type_filter` to collect tags by default.

**Comment by ChrsMark** [comment]
@kaiyan-sheng this is updated now.

File: x-pack/metricbeat/module/aws/dynamodb/_meta/docs.asciidoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+For all other DynamoDB metrics, the aggregation granularity is five minutes.
+
+[float]
+=== Configuration example
+[source,yaml]
+----
+- module: aws
+  period: 300s
+  metricsets:
+    - sns
```
Comment (kaiyan-sheng) [inline x-pack/metricbeat/module/aws/dynamodb/_meta/docs.asciidoc]: dynamodb instead

Comment (ChrsMark) [reply x-pack/metricbeat/module/aws/dynamodb/_meta/docs.asciidoc]: 🙈

**Pull request merged**